### ISSUE TITLE
Option for Closed Captions

### ISSUE THREAD

**Issue by Alicyl** [fgl27/smarttv-twitch#271]
For people who have trouble hearing, or following along with what's said, but do better with reading, I think an option for subtitles would be extremely helpful. Myself, I have trouble following voices when there's a lot of noises suppressing the voices or if they're similarly not clear/loud enough, and I have to often switch to the vanilla/modded Twitch applications for certain streams that offer the option.

I saw that you [once explored this](https://github.com/fgl27/smarttv-twitch/issues/243#issuecomment-1216658682) due to a complaint—issue, rather, that involved the subtitles appearing against the reporter's wishes, but that was a year ago. Has anything changed since then that made turning this into a toggleable feature possible?

**Comment by fgl27** [comment]
CC is it enabled across all streams or is it something only a few streams have enabled?

If I have some examples of channels that have will be easier to add a enable/disable button

**Comment by Alicyl** [comment]
I wish it was all streams, but only certain streams have them as it is entirely up to the broadcaster to go through their Creator Dashboard to add them via a Twitch extension (Closed Captioner/Verbit).

[Critical Role](https://www.twitch.tv/criticalrole) is an example of a Twitch stream I watch every Thursday that has closed captions. Because there is a lot of laughing, voices talking over each other from time to time, and music, it can sometimes be hard to follow what they're saying unless I switch to the official/a different modded Twitch application (like Purple TV) or YouTube for the subtitles.

**Comment by fgl27** [comment]
![image](https://github.com/fgl27/smarttv-twitch/assets/13755892/c626a87c-1393-4bf7-bce8-d9605bc59997)

can you share a channel that allows anyone to see their vod, as vods are always there live may not be I don't have much time to work on features if I have to choose a day and time I will not even try to add this.

**Comment by Alicyl** [comment]
> can you share a channel that allows anyone to see their vod

[Sure](https://www.twitch.tv/mish/), I forgot they locked their VODs. If you want to find more yourself outside the ones in my list, go to Browse -> Live Channels -> Type "CC" or "ClosedCaptions".

> as vods are always there live may not be I don't have much time to work on features if I have to choose a day and time I will not even try to add this.

As mentioned earlier, Closed Captions are available for both live and for VODs. Certainly, it's entirely your decision regarding rather you would(n't) mind partitioning your time to work on something like this. 

I just wanted to bring this suggestion to your attention for those with hearing/listening difficulties or disabilities that use your application since there isn't a clear way to send you feedback and suggestions outside GitHub like there is with official applications or those Discord servers (e.g., PurpleTV).
